src/reconcile.py

```diff
@@ -224,6 +229,75 @@ def _local_and_server(
     )
 
 
+def _local_and_server(
```

Comment:
I don't find this function name very descriptive

Reply:
What if it is looked at in context of where it is used in the `_calculate_action` function? An additional note, the name of this function wasn't actually changed in this PR (git is confusing the rename with a new function being added)